### DISCUSSION TITLE
Menu building with `belongs_to` and `navigation_menu` in v4

### DISCUSSION THREAD

**Discussion by kyrnich** [activeadmin/activeadmin#8239] [Help]
Hey!

I have an issue building my menu using [documented](https://activeadmin.info/2-resource-customization.html#belongs-to) `belongs_to` and `navigation_menu` methods in v4.

```
app/admin/parent.rb
ActiveAdmin.register Parent do
...
end

app/admin/child.rb
ActiveAdmin.register Child do
  belongs_to :parent
  navigation_menu :parent
  ...
end
```

From what I can see, the menu is not affected at all by using those two methods.

Am I doing something wrong? or are these methods broken for v4?

**Comment by javierjulio** [comment]
@kyrnich sorry, I don’t use this feature. It’s hard to have coverage for everything in our development app so I wouldn’t be surprised if it is. It would help if you are able to update our development app (contributing guide) with what it would take to implement. I can then take those template changes and apply to a v3 dev app to see what it renders and figure it out. Ideally, though would be best if you are able to debug and suggest/contribute an update if you depend on the feature. We do have a sample resource using belongs_to (although not navigation_menu) in the development app so I figured it was working as is. It must not be.

**Reply by kyrnich** [reply]
Got it. I'm trying `active_admin` for the first time and trying to understand what it can do. 

Would love to help with this one, but can we clarify the behavior? 

Is `navigation_menu :parent` supposed to produce a dropdown menu or a cascaded menu item with parent at the top and child under it?

```
- Parent
  - Child   
```

**Reply by javierjulio** [reply]
I don't know. That's why I suggest trying with the development app we generate when working on ActiveAdmin locally, but on the 3-0-stable branch to see what it would render previously.

**Reply by kyrnich** [reply]
It seems that it did not work in the 3-0-stable as well. At least, I don't see any noticeable changes on the page: no sidebars, dropdowns, or anything.
